client/src/layout/Layout/Layout.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { Outlet } from 'react-router-dom';
+import Footer from '../Footer/Footer';
 
 const Layout = () => (
   <>
     <header>Header</header>
     <main>
       <Outlet />
     </main>
-    <footer>Footer</footer>
```

Comment:
wrong line with footer tag

Reply:
this one is also added  = ) 

![image](https://user-images.githubusercontent.com/112622259/198880800-c261f971-3ece-44f2-978a-e2c09344f826.png)